src/engraving/types/types.h

```diff
@@ -758,7 +758,7 @@ enum class TextStyleType {
     COMPOSER,
     LYRICIST,
     TRANSLATOR,
-    FRAME,
+    BORDER,
```

Comment:
This one should be dropped too (it refers to the text style of text used in "text frames")